src/components/About/Blog/index.js

```diff
@@ -381,24 +439,92 @@ class Blog extends Component {
                               {posts.author}
                             </a>
                           </CustomTypography>
-                          <Typography variant="body1" gutterBottom>
-                            {htmlContent}
-                          </Typography>
-                        </BlogPostContainer>
+                        </CardContent>
+                        <CardMedia
+                          image={image}
+                          style={{
+                            height: '20rem',
+                            margin: '0 auto',
+                          }}
+                        />
+                        <Overlay>
+                          <OverlayLink href={posts.link}>
+                            {`Published on ${dateFormat(
+                              d,
+                              'dddd, mmmm dS, yyyy',
+                            )}`}
+                          </OverlayLink>
+                        </Overlay>
+                        {this.state.expandedElements.includes(i) ? (
+                          <Fade in={true} timeout={1100}>
```

Comment:
Please refactor out each Blog card into a separate file and have the expanded logic inside that file.